### ISSUE TITLE
大部分的激活函数都在 paddle.nn.functional, 为什么 tanh 可以直接 paddle.tanh

### ISSUE THREAD

**Issue by yt605155624** [PaddlePaddle/Paddle#37615]
大部分的激活函数都在 paddle.nn.functional 里面, 比如 sigmoid softmax，但是可以直接通过 paddle.tanh 调用 tanh，感觉这样接口很不统一 。。

**Comment by zhangting2020** [comment]
您好，感谢您的建议，Paddle中对于Tensor的基础运算通过paddle.xx进行调用，您反馈的tanh和其他的例如cosh、cos等API被归类为基础的数学操作。尽管它们同时也可以用作激活函数。

**Comment by Ligoml** [comment]
事实上 paddle 也是支持 paddle.nn.functional.tanh() 的调用方式的~你可以自己测试下
